2023-03-15 23:13:08,283 INFO    MainThread:1349 [wandb_setup.py:_flush():76] Configure stats pid to 1349
2023-03-15 23:13:08,283 INFO    MainThread:1349 [wandb_setup.py:_flush():76] Loading settings from /home/seuyon0101/.config/wandb/settings
2023-03-15 23:13:08,283 INFO    MainThread:1349 [wandb_setup.py:_flush():76] Loading settings from /home/seuyon0101/aiffel/saturi/notebook/Model/wandb/settings
2023-03-15 23:13:08,283 INFO    MainThread:1349 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-03-15 23:13:08,284 INFO    MainThread:1349 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-03-15 23:13:08,284 INFO    MainThread:1349 [wandb_init.py:_log_setup():506] Logging user logs to /home/seuyon0101/aiffel/saturi/notebook/Model/wandb/run-20230315_231308-x73a9bni/logs/debug.log
2023-03-15 23:13:08,284 INFO    MainThread:1349 [wandb_init.py:_log_setup():507] Logging internal logs to /home/seuyon0101/aiffel/saturi/notebook/Model/wandb/run-20230315_231308-x73a9bni/logs/debug-internal.log
2023-03-15 23:13:08,284 INFO    MainThread:1349 [wandb_init.py:init():546] calling init triggers
2023-03-15 23:13:08,284 INFO    MainThread:1349 [wandb_init.py:init():552] wandb.init called with sweep_config: {}
config: {'model_name': 'Vanilla_Transformer_bpe_40009', 'n_layers': 6, 'd_model': 512, 'n_heads': 8, 'd_ff': 2048, 'src_vocab_size': 4009, 'tgt_vocab_size': 4009, 'pos_len': 256, 'dropout': 0.2, 'shared': True, 'batch_size': 32, 'warmups': 4000, 'epochs': 1, 'optimizer': 'ADAM', 'loss': 'SparseCategoricalCrossentropy', 'metric': 'bleu'}
2023-03-15 23:13:08,285 INFO    MainThread:1349 [wandb_init.py:init():568] re-initializing run, found existing run on stack: 1fii0wqm
2023-03-15 23:13:08,287 INFO    MainThread:1349 [wandb_run.py:_finish():1852] finishing run saturi/PRJ_saturi_model_exp1/1fii0wqm
2023-03-15 23:13:08,287 INFO    MainThread:1349 [jupyter.py:save_history():445] not saving jupyter history
2023-03-15 23:13:08,288 INFO    MainThread:1349 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2023-03-15 23:13:08,288 INFO    MainThread:1349 [wandb_init.py:_jupyter_teardown():434] cleaning up jupyter logic
2023-03-15 23:13:08,288 INFO    MainThread:1349 [wandb_run.py:_atexit_cleanup():2081] got exitcode: 0
2023-03-15 23:13:08,288 INFO    MainThread:1349 [wandb_run.py:_restore():2064] restore
2023-03-15 23:13:08,289 INFO    MainThread:1349 [wandb_run.py:_restore():2070] restore done
2023-03-15 23:13:11,893 INFO    MainThread:1349 [wandb_run.py:_footer_history_summary_info():3429] rendering history
2023-03-15 23:13:11,894 INFO    MainThread:1349 [wandb_run.py:_footer_history_summary_info():3461] rendering summary
2023-03-15 23:13:11,896 INFO    MainThread:1349 [wandb_run.py:_footer_sync_info():3387] logging synced files
2023-03-15 23:13:11,901 INFO    MainThread:1349 [wandb_init.py:init():602] starting backend
2023-03-15 23:13:11,901 INFO    MainThread:1349 [wandb_init.py:init():606] setting up manager
2023-03-15 23:13:11,907 INFO    MainThread:1349 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-03-15 23:13:11,910 INFO    MainThread:1349 [wandb_init.py:init():613] backend started and connected
2023-03-15 23:13:11,923 INFO    MainThread:1349 [wandb_run.py:_label_probe_notebook():1202] probe notebook
2023-03-15 23:13:11,923 INFO    MainThread:1349 [wandb_run.py:_label_probe_notebook():1212] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-03-15 23:13:11,923 INFO    MainThread:1349 [wandb_init.py:init():701] updated telemetry
2023-03-15 23:13:12,155 INFO    MainThread:1349 [wandb_init.py:init():741] communicating run to backend with 60.0 second timeout
2023-03-15 23:13:13,034 INFO    MainThread:1349 [wandb_run.py:_on_init():2131] communicating current version
2023-03-15 23:13:13,099 INFO    MainThread:1349 [wandb_run.py:_on_init():2140] got version response upgrade_message: "wandb version 0.14.0 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2023-03-15 23:13:13,099 INFO    MainThread:1349 [wandb_init.py:init():789] starting run threads in backend
2023-03-15 23:13:17,720 INFO    MainThread:1349 [wandb_run.py:_console_start():2112] atexit reg
2023-03-15 23:13:17,720 INFO    MainThread:1349 [wandb_run.py:_redirect():1967] redirect: SettingsConsole.WRAP_RAW
2023-03-15 23:13:17,721 INFO    MainThread:1349 [wandb_run.py:_redirect():2032] Wrapping output streams.
2023-03-15 23:13:17,721 INFO    MainThread:1349 [wandb_run.py:_redirect():2057] Redirects installed.
2023-03-15 23:13:17,722 INFO    MainThread:1349 [wandb_init.py:init():831] run started, returning control to user process
2023-03-15 23:20:15,825 INFO    MainThread:1349 [wandb_setup.py:_flush():76] Configure stats pid to 1349
2023-03-15 23:20:15,825 INFO    MainThread:1349 [wandb_setup.py:_flush():76] Loading settings from /home/seuyon0101/.config/wandb/settings
2023-03-15 23:20:15,825 INFO    MainThread:1349 [wandb_setup.py:_flush():76] Loading settings from /home/seuyon0101/aiffel/saturi/notebook/Model/wandb/settings
2023-03-15 23:20:15,825 INFO    MainThread:1349 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-03-15 23:20:15,825 INFO    MainThread:1349 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-03-15 23:20:15,826 INFO    MainThread:1349 [wandb_init.py:_log_setup():506] Logging user logs to /home/seuyon0101/aiffel/saturi/notebook/Model/wandb/run-20230315_232015-yhz3cb5j/logs/debug.log
2023-03-15 23:20:15,826 INFO    MainThread:1349 [wandb_init.py:_log_setup():507] Logging internal logs to /home/seuyon0101/aiffel/saturi/notebook/Model/wandb/run-20230315_232015-yhz3cb5j/logs/debug-internal.log
2023-03-15 23:20:15,826 INFO    MainThread:1349 [wandb_init.py:_jupyter_setup():452] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7fea1c54b2b0>
2023-03-15 23:20:15,826 INFO    MainThread:1349 [wandb_init.py:init():546] calling init triggers
2023-03-15 23:20:15,827 INFO    MainThread:1349 [wandb_init.py:init():552] wandb.init called with sweep_config: {}
config: {'model_name': 'Vanilla_Transformer_bpe_40009', 'n_layers': 6, 'd_model': 512, 'n_heads': 8, 'd_ff': 2048, 'src_vocab_size': 4009, 'tgt_vocab_size': 4009, 'pos_len': 23, 'dropout': 0.2, 'shared': True, 'batch_size': 32, 'warmups': 4000, 'epochs': 1, 'optimizer': 'ADAM', 'loss': 'SparseCategoricalCrossentropy', 'metric': 'bleu'}
2023-03-15 23:20:15,827 INFO    MainThread:1349 [wandb_init.py:init():568] re-initializing run, found existing run on stack: x73a9bni
2023-03-15 23:20:15,829 INFO    MainThread:1349 [wandb_run.py:_finish():1852] finishing run saturi/PRJ_saturi_model_exp1/x73a9bni
2023-03-15 23:20:15,829 INFO    MainThread:1349 [wandb_run.py:_atexit_cleanup():2081] got exitcode: 0
2023-03-15 23:20:15,829 INFO    MainThread:1349 [wandb_run.py:_restore():2064] restore
2023-03-15 23:20:15,829 INFO    MainThread:1349 [wandb_run.py:_restore():2070] restore done
2023-03-15 23:20:22,028 INFO    MainThread:1349 [wandb_run.py:_footer_history_summary_info():3429] rendering history
2023-03-15 23:20:22,028 INFO    MainThread:1349 [wandb_run.py:_footer_history_summary_info():3461] rendering summary
2023-03-15 23:20:22,031 INFO    MainThread:1349 [wandb_run.py:_footer_sync_info():3387] logging synced files
